ideascube/views.py

```diff
@@ -241,6 +241,25 @@ def get_row(self, user):
 user_export = staff_member_required(UserExport.as_view())
 
 
+class UserImport(FormView):
+    form_class = UserImportForm
+    template_name = 'ideascube/user_import.html'
+    success_url = reverse_lazy('user_import')
+
+    def form_valid(self, form):
+        users, errors = form.save()
+        if users:
+            msg = _('Successfully processed {count} users.')
+            msg = msg.format(count=len(users))
+            messages.add_message(self.request, messages.SUCCESS, msg)
+        if errors:
+            for error in errors:
```

Comment:
You can drop the `if errors` line. If `errors` is empty, then the `for` loop just won't do anything.